pennylane/pulse/parametrized_evolution.py

```diff
@@ -320,11 +322,11 @@ def circuit(param, time):
 
         >>> circuit(param, time)
         Array([[1.        , 0.        ],
-               [0.9897738 , 0.01022595],
-               [0.9599043 , 0.04009585],
-               [0.9123617 , 0.08763832],
-               [0.84996957, 0.15003097],
-               [0.7761489 , 0.22385144]], dtype=float32)
+            [0.98977406, 0.01022594],
+            [0.95990416, 0.04009584],
+            [0.91236167, 0.08763833],
+            [0.84996865, 0.15003133],
+            [0.77614817, 0.22385181]], dtype=float64)
```

Comment:
Why was this formatting changed? Running this locally, the old formatting seems consistent with what is being displayed.